store/sqlstore/file_info_store.go

```diff
@@ -625,11 +624,32 @@ func (fs SqlFileInfoStore) Search(paramsList []*model.SearchParams, userId, team
 				excludeClause = " & !(" + strings.Join(strings.Fields(excludedTerms), " | ") + ")"
 			}
 
+			exactPhraseTerms := []string{}
+
+			if exactPhraseRegExp, exactPhraseRegExpErr := regexp.Compile(`".+"\s*`); exactPhraseRegExpErr == nil {
```

Comment:
I fixed the regular expression. Thanks for the suggestion. 
I removed the last part(`\s*`) because the spaces will be erased when we use `strings.Fields` in the next line.